src/test/java/apoc/date/DateTest.java

```diff
@@ -0,0 +1,119 @@
+package apoc.date;
+
+import java.text.SimpleDateFormat;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.List;
+import java.util.stream.Stream;
+import apoc.util.TestUtil;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.neo4j.graphdb.GraphDatabaseService;
+import org.neo4j.graphdb.Node;
+import org.neo4j.graphdb.Transaction;
+import org.neo4j.helpers.collection.Iterators;
+import org.neo4j.test.TestGraphDatabaseFactory;
+import static apoc.util.TestUtil.testCall;
+import static java.util.stream.Collectors.toList;
+import static org.junit.Assert.*;
+
+
+public class DateTest {
+	private GraphDatabaseService db;
+
+	@Before
+	public void setUp() throws Exception {
+		db = new TestGraphDatabaseFactory().newImpermanentDatabase();
+		TestUtil.registerProcedure(db, Date.class);
+	}
+
+	@After
+	public void tearDown() {
+		db.shutdown();
+	}
+
+	@Test public void testDateAccessor() throws Exception {
+		try (Transaction tx = db.beginTx()) {
```

Comment:
you can simplify the test by just passing in the string value a parameter value. No need to create nodes and such.